### PR TITLE
217 add sql cmd for test

### PR DESCRIPTION
**Pull request by Palashio**
## Describe your changes

Implements logic for new test sql command. This command currently allows users to quickly return rows in a set of data that satisfy arguments passed to the magic.

## Issue ticket number and link
Closes #217 

## Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] I have added thorough [tests](https://github.com/ploomber/contributing/blob/main/CONTRIBUTING.md#testing) (when necessary).
- [ ] I have added the right documentation in the [docstring](https://github.com/ploomber/contributing/blob/main/CONTRIBUTING.md#documenting-changes-and-new-features) and [changelog](https://github.com/ploomber/contributing/blob/main/CONTRIBUTING.md#changelog) (when needed)

----
:books: Documentation preview :books:: https://jupysql--258.org.readthedocs.build/en/258/

### CODE REVIEW

**Comment by Palashio** [comment]
Created a new PR because the last rebase went bad.

**Review by tonykploomber — CHANGES_REQUESTED** [review]
Code section LGTM

@edublancas 

We are missing the doc for this `%sqlcmd test` now, where do you think we should put the doc for this?

Option 1. Add one more section in [tables-columns](https://jupysql.ploomber.io/en/latest/user-guide/tables-columns.html)
Option 2. Add one more page under `user-guide`

File: src/tests/integration/test_generic_db_opeations.py [hunk 1]
```diff
@@ -144,3 +144,32 @@ def test_telemetry_execute_command_has_connection_info(
             },
         },
     )
+
+
+def test_sql_cmd_magic_uno(ip):
```
Comment (tonykploomber) [inline src/tests/integration/test_generic_db_opeations.py:149]: Actually we need the parametrized ips to enable the integration test for different databases

```
@pytest.mark.parametrize(
    "ip_with_dynamic_db",
    [
        ("ip_with_postgreSQL"),
        ("ip_with_mySQL"),
        ("ip_with_mariaDB"),
        ("ip_with_SQLite"),
        ("ip_with_duckDB"),
    ],
)
def test_sql_cmd_magic_uno(ip_with_dynamic_db, request):
    ip_with_dynamic_db = request.getfixturevalue(ip_with_dynamic_db)
    ...
```

**Review by tonykploomber — CHANGES_REQUESTED** [review]
[This is](https://github.com/ploomber/jupysql/issues/248#issuecomment-1472483937) how to setup the local env to init. integration testing environment

**Pull request closed without merging**